### PR TITLE
Success message popups

### PR DESCRIPTION
**Pull request by t-wing11**
Included success popup messages for the following modals

Upload modal - "Files Uploaded"
Create Graph Modal - "Graph Created"
Create Graph Modal - "Series Created"

Added a styling change for the "x" button next the files that will be uploaded

### CODE REVIEW

File: front-end/src/components/AnalyzersAndSeries.js [hunk 1]
```diff
@@ -2,9 +2,8 @@ import '../styles/modalStyles.css';
 import Help from './Help';
 import analyzerData from './analyzerData';
 import '../styles/analyzersAndSeriesStyles.css';
-import React, { useState } from 'react';
-
-const AnalyzersAndSeries = ({ dimensions, columns, setDisplayPage, setShowModal, handleSubmit }) => {
+import { useState } from 'react';
+const AnalyzersAndSeries = ({ setSuccessMessage,success,dimensions, columns, setDisplayPage, setShowModal, handleSubmit }) => {
```
Comment (gr812b) [inline front-end/src/components/AnalyzersAndSeries.js:6]: In the future, please just add it at the end. Easier to follow

File: front-end/src/components/AnalyzersAndSeries.js [hunk 2]
```diff
@@ -24,12 +23,20 @@ const AnalyzersAndSeries = ({ dimensions, columns, setDisplayPage, setShowModal,
...
         for (let i = 0; i < dimensions; i++) {
             selectColumns.push(JSON.parse(document.getElementsByClassName(i)[0].value));
         }
+        if (isSeries) {
+            setSuccessMessage("Series Added");
+            success();
+        } else {
+            setSuccessMessage("Graph Created");
+            success();
+        }
```
Comment (gr812b) [inline front-end/src/components/AnalyzersAndSeries.js:39]: success is called either way, can just be called once after the if else

File: front-end/src/components/App.js [hunk 3]
```diff
@@ -55,13 +58,19 @@ const App = () => {
     type: "line"
   });
 
+  const [successMessage, setSuccessMessage] = useState("")
+
+  const success = () => {
+    $( "div.success" ).fadeIn(1000).delay(2000).fadeOut(1000)
+  };
```
Comment (gr812b) [inline front-end/src/components/App.js:65]: Didn't find anything that looked cooler than fade in?

File: front-end/src/styles/App.css [hunk 4]
```diff
@@ -68,3 +68,31 @@ button {
 button:hover {
   background-color: rgb(238, 48, 86);
 }
+
+select {
+  width: 200px;
+  padding: 5px;
+  font-size: 12px;
+}
```
Comment (gr812b) [inline front-end/src/styles/App.css:76]: Why is there select styling here?

File: front-end/src/styles/uploadModalStyles.css [hunk 5]
```diff
@@ -82,4 +82,11 @@
   background: crimson;
   color: white;
   cursor: pointer;
+}
+
+.fileButton{
+  color: white;
+  background-color: red;
+  margin: 0;
+  padding: 7px 7px;
```
Comment (gr812b) [inline front-end/src/styles/uploadModalStyles.css:91]: What is this?

**Review by gr812b — CHANGES_REQUESTED** [review]

File: front-end/src/components/AnalyzersAndSeries.js (hunk 1 above)
Comment (gr812b) [reply front-end/src/components/AnalyzersAndSeries.js:6]: Also, use spaces! Format correctly pls

**Comment by gr812b** [comment]
Im willing to bet we can avoid the success function being passed and just use a useEffect function, will test and see

File: front-end/src/styles/uploadModalStyles.css (hunk 5 above)
Comment (t-wing11) [reply front-end/src/styles/uploadModalStyles.css:91]: its the x button next to the files that you upload, without it the button is massive and looks super dumb

File: front-end/src/components/AnalyzersAndSeries.js (hunk 1 above)
Comment (t-wing11) [reply front-end/src/components/AnalyzersAndSeries.js:6]: https://www.youtube.com/watch?v=unccXlME1X4

File: front-end/src/styles/App.css (hunk 4 above)
Comment (t-wing11) [reply front-end/src/styles/App.css:76]: I have no idea, trying to find where it was added in and cant find it. Either way I removed it and nothing changed

File: front-end/src/components/AnalyzersAndSeries.js (hunk 2 above)
Comment (t-wing11) [reply front-end/src/components/AnalyzersAndSeries.js:39]: Yup u right

**Comment by gr812b** [comment]
I also found a bug in this, that updating state clears variables that aren't state. This meant that adding series would not work

File: front-end/src/components/App.js (hunk 3 above)
Comment (t-wing11) [reply front-end/src/components/App.js]: I actually dont think there is another option with jquery. At least none that I found. Personally I like it so I dont see the need to change anything

**Review by gr812b — APPROVED** [review]
Looks great

**Pull request merged**